test/leaks.js

```diff
@@ -19,25 +27,48 @@ const internals = {
 const lab = exports.lab = _Lab.script();
 const describe = lab.describe;
 const it = lab.it;
+const afterEach = lab.afterEach;
+const beforeEach = lab.beforeEach;
 const expect = Code.expect;
 
 
 describe('Leaks', () => {
 
+    let testedKeys = [];
+
+    beforeEach((done) => {
+
+        testedKeys = [];
+        done();
+    });
+
+    afterEach((done) => {
+
+        testedKeys.forEach((key) => {
+            // Only delete globals that were manually set, and avoid deleting pre-existing globals
+            if (global[testedKeys] && global[testedKeys] === 1) {
+                delete global[testedKeys];
+            }
+        });
+        done();
+    });
```

Comment:
On Windows, `global` keys can actually be deleted, which makes those test not runnable.
I refactored them to:
- automatically delete globals set to 1, and *not* delete the pre-existing values
- check all counter globals existence and absence.